src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -1,376 +1,167 @@
-            collapsible: true,
-            active     : true,
+        const modal_back_items = document.getElementsByClassName('modal-back');
+        Array.from(modal_back_items).forEach((item) => {
+            item.addEventListener('click', () => {
+                hideDialogs();
+            });
         });
-        const $account_closure_warning = $('#account_closure_warning');
-        const $account_closure_error = $('#account_closure_error');
 
-        const hideDialogs = () => {
-            $account_closure_warning.setVisibility(0);
-            $account_closure_error.setVisibility(0);
-        };
+        el_dialog_container.setVisibility(1);
 
-        hideDialogs();
-
-        $('.back').on('click', () => {
-            hideDialogs();
+        el_deacivate_button.addEventListener('click', () => {
+            el_account_closure_warning.setVisibility(0);
+            deactivate();
         });
 
-        $('#deactivate').on('click', () => {
-            $account_closure_warning.setVisibility(0);
-            submitForm($account_closure_error);
+        el_form_closure_step_1.addEventListener('submit', (e) => {
+            e.preventDefault();
+            showStep(2);
         });
 
-        $(form_selector).on('submit', (event) => {
-            event.preventDefault();
-            if (getReason()) {
-                $account_closure_warning.setVisibility(1);
+        el_step_2_submit.addEventListener('click', (e) => {
+            if (!el_step_2_submit.classList.contains('button-disabled')) {
+                e.preventDefault();
+                el_account_closure_warning.setVisibility(1);
             }
         });
 
-        $txt_other_reason.setVisibility(0);
+        el_step_2_back.addEventListener('click', () => {
+            showStep(1);
+            $.scrollTo(0, 500);
+        });
 
-        $txt_other_reason.on('keyup', () => {
-            const input = $txt_other_reason.val();
-            if (input && validateReasonTextField(false)) {
-                $txt_other_reason.removeClass('error-field');
-                $error_msg.css('display', 'none');
-            }
+        reason_checkbox_list.forEach(element => {
+            element.addEventListener('change', () => { onSelectedReasonChange(); });
         });
-        $('#reason input[type=radio]').on('change', (e) => {
-            const { value } = e.target;
 
-            if (value === 'other') {
-                $txt_other_reason.setVisibility(1);
-            } else {
-                $txt_other_reason.setVisibility(0);
-                $txt_other_reason.removeClass('error-field');
-                $txt_other_reason.val('');
-                $error_msg.css('display', 'none');
-            }
+        el_suggested_improves.addEventListener('input', onTextChanged);
+        el_other_trading_platforms.addEventListener('input', onTextChanged);
+    };
+
+    const showStep = (step) => {
+        Array.from(new Array(3)).forEach((_, index) => {
```

Comment:
perhaps storing 3 as a constant  above the onload function as the total number of steps